hal/src/thumbv7em/clock/v2/osculp32k.rs

```diff
@@ -103,37 +93,19 @@ impl OscUlp32kToken {
 /// It is generic over:
 /// - An output state of a 32 kHz signal (active/inactive)
 /// - An output state of a 1 kHz signal (active/inactive)
-pub struct OscUlp32k<X, Y>
-where
-    X: Output32k,
```

Comment:
The same here, multiplexer switches must be reflected in types somehow. Otherwise it will be unsound.